src/components/Calculator.js

```diff
@@ -9,66 +9,69 @@ function Calculator() {
   };
 
   return (
-    <div className="clac-container">
-      <div className="result">{state.next || state.total || 0}</div>
-      <button type="button" className="btn" onClick={handleClick}>
-        AC
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        +/-
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        %
-      </button>
-      <button type="button" className="btn btn-orange" onClick={handleClick}>
-        /
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        7
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        8
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        9
-      </button>
-      <button type="button" className="btn btn-orange" onClick={handleClick}>
-        x
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        4
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        5
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        6
-      </button>
-      <button type="button" className="btn btn-orange" onClick={handleClick}>
-        -
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        1
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        2
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        3
-      </button>
-      <button type="button" className="btn btn-orange" onClick={handleClick}>
-        +
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        0
-      </button>
-      <button type="button" className="btn" onClick={handleClick}>
-        .
-      </button>
-      <button type="button" className="btn btn-orange" onClick={handleClick}>
-        =
-      </button>
-    </div>
+    <>
+      <h2 className="heading">Let&apos;s do some math!</h2>
+      <div className="clac-container">
+        <div className="result">{state.next || state.total || 0}</div>
+        <button type="button" className="btn" onClick={handleClick}>
+          AC
+        </button>
+        <button type="button" className="btn" onClick={handleClick}>
+          +/-
+        </button>
+        <button type="button" className="btn" onClick={handleClick}>
+          %
+        </button>
+        <button type="button" className="btn btn-orange" onClick={handleClick}>
+          /
+        </button>
+        <button type="button" className="btn" onClick={handleClick}>
+          7
+        </button>
+        <button type="button" className="btn" onClick={handleClick}>
+          8
+        </button>
+        <button type="button" className="btn" onClick={handleClick}>
+          9
```

Comment:
- [OPTIONAL] 🤔 This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: 👍🏻